tool_commands/command_box.py

```diff
@@ -114,25 +108,22 @@ def run_script(self, qm_idx):
         # We're only interested in leaves of the tree:
         # TODO: need to make sure the leaf is not an empty directory
         if self.is_leaf(qm_idx):
-            filename = qm_idx.data()
-            item = self.commandboxmodel.item(qm_idx.row(), qm_idx.column())
-            path = self.leaf_path(qm_idx)
-
-            logger.debug(filename)
-            logger.debug(item)
-            logger.debug(path)
-
-            curr_dir = os.path.dirname(__file__)
-            module_path = os.path.join(curr_dir, *path)
-            name, ext = os.path.splitext(path[-1])
+            module_name = qm_idx.data()
+            package_name = modulename_packagename_mapping.get(module_name)
+            if not package_name:
+                logging.warning("package of clicked command not found")
+                return
+            tool_commands_dir = Path(__file__).parent
+            module_path = tool_commands_dir / package_name / module_name
+            logger.debug(module_name)
+            logger.debug(module_path)
+            name = module_path.stem
+            ext = module_path.suffix
             if ext != ".py":
                 logger.error("Not a Python script")
                 return
 
-            logger.debug(module_path)
-            logger.debug(name)
-
-            loader = SourceFileLoader(name, module_path)
+            loader = SourceFileLoader(name, str(module_path))
```

Comment:
Willen we later gewoon "import" kunnen gebruiken? Zou op zich moeten kunnen, toch? Als de spaties in underscores zijn veranderd?

Reply:
>>> import importlib
>>> importlib.import_module(str(module_path))
ModuleNotFoundError: No module named '/home/nens/'

Reply:
dotted ipv path